www/main/navigator/shabad/ShabadContent.jsx

```diff
@@ -64,6 +64,10 @@ const ShabadContent = () => {
   };
 
   const filterRequiredVerseItems = verses => {
+		if (verses.flat) {
+			verses = verses.flat(1);
```

Comment:
This condition `if(verses.flat)` will always be true. This if condition is equivalent to `if ([])` which will always run. Please correct me if I am wrong.

Reply:
It does not, sometimes it gets an error, because verses is a realm object, and it doesnt have a flat function.

Reply:
I think the ideal way to handle that realm exception could be using a `try` over there. Because writing the if statement using array method and not calling is doesn't look right. Do you think we should handle this by using `try` veerji?